scripts/busybox-ping-test/build_aca.sh

```diff
@@ -0,0 +1,160 @@
+#! /bin/sh
+
+# Pull latest ACA code from specified repository, branch and commit
+# and call aca_machine_init.sh to do the actual build.
+# Emit success or failure so that the remote script learn about the status
+# of the build.
+
+
+GIT_REPO=""
+GIT_BRANCH=""
+GIT_COMMIT=
+GIT_URL=
+REMOTE_COMMIT=
+LOCAL_COMMIT=
+DO_FORCE=0
+DO_BUILD=0
+
+# Get local repo/branch/commit info
+git_check_current() {
+    LOCAL_INFO=`git log origin/$GIT_BRANCH | head -3`
+    LOCAL_COMMIT=`echo $LOCAL_INFO | awk '/^commit/ {print $2}'`
+    LOCAL_DATE=`echo $LOCAL_INFO | sed 's/^.*Date://' | awk '{print $1, $2, $3, $4, $5}'`
+}
+
+
+# Get remote repo/branch/commit info
+# May need git fetch -all to be robust
+git_check_remote() {
+    REMOTE_INFO=`git log origin/$GIT_BRANCH | head -3`
+    REMOTE_COMMIT=`echo $REMOTE_INFO | awk '/^commit/ {print $2}'`
+    REMOTE_DATE=`echo $REMOTE_INFO |  sed 's/^.*Date://' | awk '{print $1, $2, $3, $4, $5}'`
+
+    RSEC=`date --date="${REMOTE_DATE}" +%s`
```

Comment:
Same date might have multiple commits. What do we do with that? :-)

Reply:
Full dates are converted into seconds since epoch before comparing. It would not be possible for two commits have same timestamp down to the last second since each commit effectively locks the repository until it is done. So we are fine.

Reply:
Cool that is what we need exactly.